libs/datasets/manual_filter.py

```diff
@@ -0,0 +1,111 @@
+import dataclasses
+
+import structlog
+from covidactnow.datapublic.common_fields import CommonFields
+from covidactnow.datapublic.common_fields import PdFields
+import pandas as pd
+
+from libs.datasets import AggregationLevel
+from libs.datasets import taglib
+from libs.datasets import timeseries
+from libs.pipeline import Region, RegionMask
+
+
+_logger = structlog.getLogger()
+
+
+# TODO(tom): Make some kind of hierarchy of class to form a schema that can be populated by a JSON.
+CONFIG = {
+    "filters": [
+        {
+            "regions_included": [
+                Region.from_fips("49009"),
+                Region.from_fips("49013"),
+                Region.from_fips("49047"),
+            ],
+            "drop_observations": [
```

Comment:
Renamed and removed list. I like the idea of having some kind of region selector and then one or more filter steps associated because it matches what is implemented.